pipeline/terraform/stack/service_image_ingestor.tf

```
@@ -61,6 +62,15 @@ module "ingestor_images" {
   shared_logging_secrets = var.shared_logging_secrets
 }
 
+module "image_ingestor_topic" {
+  source = "../modules/topic"
+
+  name       = "${local.namespace_hyphen}_image_ingestor"
```

```suggestion
  name       = "${local.namespace_hyphen}_image_ingestor_output"
```

I know it being an output is implied by it being an SNS topic, but I find it useful to distinguish between topics that are the output of a service, and the result of something like a script.